examples/demo/basic/discrete_cmap_image_plot.py

```diff
@@ -18,7 +18,7 @@
 from traitsui.api import Item, Group, View
 
 # Chaco imports
-from chaco.api import ArrayPlotData, jet, Plot
+from chaco.api import ArrayPlotData, viridis, Plot
```

Comment:
We're not even using this color map in this example - looks like we're using the `accent` import from `chaco.default_colormaps`.

The example looks very weird with the `accent` color map - it looks much better with `viridis`.